Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -1348,26 +1348,3 @@
     Telecrystal: 1
   categories:
     - UplinkPointless
-
-- type: listing
-  id: UplinkSyndicatePersonalAI
-  name: uplink-syndicate-pai-name
-  description: uplink-syndicate-pai-desc
-  icon: { sprite: /Textures/Objects/Fun/pai.rsi, state: syndicate-icon-pai-off }
-  productEntity: SyndicatePersonalAI
-  cost:
-    Telecrystal: 1
-  categories:
-    - UplinkPointless
-  conditions:
-  - !type:ListingLimitedStockCondition
-    stock: 1
-- type: listing
```

Comment:
You have ripped out the Syndicate pAI along with your suit and tie

Reply:
fuck